data_collection/gazette/spiders/ba_santa_rita_de_cassia.py

```diff
@@ -0,0 +1,9 @@
+from gazette.spiders.base.portal_da_transparencia import PortalDaTransparenciaBaseSpider
+
+
+class BaSantaRitadeCassiaSpider(PortalDaTransparenciaBaseSpider):
+    TERRITORY_ID = "2928406"
+    name = "ba_santa_rita_de_cassia"
+    start_urls = [
+        "http://ba.portaldatransparencia.com.br/prefeitura/santaritadecassia/"
```

Comment:
The last edition dates from 31/Dezembro/2020. Could you take a look at the current gazettes' website?

Reply:
They seem to still be using the platform, but with the URL https://doem.org.br/ba/santaritadecassia